config/sidebar-learn.json

```diff
@@ -394,10 +394,15 @@
 							"slug": "reduce-indexing-memory-usage"
 					},
 					{
-						"source": "learn/experimental/ranking_rule_score_details.mdx",
-						"label": "Ranking score details",
-						"slug": "ranking_rule_score_details"
-				  }
+						  "source": "learn/experimental/ranking_rule_score_details.mdx",
+						  "label": "Ranking score details",
```

Comment:
Not related to vector search but I think "Detailed ranking scores" is a clearer title for this page.